### ISSUE TITLE
Use yerbamate.dev links for images in documentation

### ISSUE THREAD

**Issue by Nutomic** [LemmyNet/lemmy#655]
Images in the documentation are currently hosted on imgur, which is not great for privacy. We should use links from yerbamate.dev instead (my gitea instance where we have a mirror of the Lemmy source).

[Discussed here before](https://github.com/LemmyNet/lemmy/pull/654#discussion_r412890778)

**Comment by niniack** [comment]
Hey, I'm interested in helping out with this. Would this involve re-uploading the images in the doc to your mirror of the Lemmy source and then replacing the links with the links from your mirror?

**Comment by Nutomic** [comment]
I guess the easiest option is to make two PRs. First make a PR to add the images to the repo, once that is merged, make another PR to replace the links. Otherwise its hard to make sure that the image links are correct.

You can make the PRs here or on Gitea, doesnt matter.

**Comment by niniack** [comment]
I think this issue is resolved! Should we close it?

**Comment by dessalines** [comment]
Yup, good catch.
